definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -1825,5 +1825,12 @@
     "CaseEventID": "linkBulkCaseReference",
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "07/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "updateBulkCaseHearingDetails",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
```

Comment:
Do you mind checking with @gavinclark if the new superuser role that I just merged needs access to this as well? (as well as the field)

Reply:
This should be triggered through the bulk case callback. The caseworker shouldn't need to use this event, though it is there just in case.

Reply:
Can probably be clarified and fixed when the actual config story for this is played (i.e. which tab to display CourtName on etc.)
Just need this for my COS Task right now